inst/extdata/gcam-usa/NREL_us_re_capacity_factors.csv

```diff
@@ -0,0 +1,56 @@
+# File: NREL_us_re_capacity_factors.csv,,,,,,
+# Title: Capacity factors by state by renewable technology,,,,,,
+# Units: Unitless,,,,,,
+State,Urban_Utility_scale_PV,Rural_Utility_scale_PV,Rooftop_PV,CSP,Onshore_Wind,Offshore_Wind
```

Comment:
I've already opened an issue. I've assigned Page, Pralit,  Aaron and myself. Page and Pralit just in case they have some idea - the file comes from a project that they worked on together several years ago.